pkg/github/service/config_loader.go

```diff
@@ -44,6 +44,16 @@ func (l *LoadableConfig) loadFromRawFile(pathTemplate string) config.Source {
 
 		l.BaseConfig.LocationURL = githubBaseURL + rawFileService.GetRelativePath(filePath)
 		l.BaseConfig.PluginName = l.PluginName
+
+		pluginHintPath := fmt.Sprintf(".ike-prow/%s_hint.md", l.PluginName)
+		hintURL := rawFileService.GetRawFileURL(pluginHintPath)
+
+		_, e := utils.GetFileFromURL(hintURL)
+
+		if e == nil {
+			l.BaseConfig.PluginHint = githubBaseURL + rawFileService.GetRelativePath(pluginHintPath)
+		}
```

Comment:
this shouldn't be here. This function is for loading configuration file, not the *_hint file. 
In addition, on the line 51 you download the file - that should be lazy. Download it (and generate the default path) only when needed. 
This should be part of `comment_message.go`